faq/index.md

```diff
@@ -66,7 +66,14 @@ alternative ways of requesting certain operations (e.g., resource creation via
 
 ## Is there a JSON Schema describing JSON API? <a href="#is-there-a-json-schema-describing-json-api" id="is-there-a-json-schema-describing-json-api" class="headerlink"></a>
 
-Not currently, no. JSON Schema cannot fully represent the semantics of JSON API, and so any such schema would be partial anyway.
+Yes, you can find the JSON Schema definition at
+[http://jsonapi.org/schema](http://jsonapi.org/schema). Please note that this
+schema is not a perfect document. Just because a JSON document may validate
+against this schema, that does not necessarily mean it is a valid JSON API
+document. The schema is provided for a base level sanity check.
```

Comment:
Following this PR, and the preceding issue #271, I agree that a schema is helpful.  If it doesn't validate all of the semantics of JSON API, then I would say it's like most schemas in that respect.  It can yield some "false positives," where a given JSON input passes schema validation, but breaks some JSON API rules. 

Still, I assume that whatever this schema does validate is in fact a requirement of the JSON API spec.  So it should not yield any "false negatives."  If a given input fails validation against this schema, then it's definitely not valid JSON API.

That's an important guarantee, and one worth stating in the FAQ, I think.  Rather than saying it's "not a perfect document," say something like "Please note that this schema only enforces a subset of the JSON API specification."